pkg/libvirt/domain.go

```diff
@@ -116,6 +119,34 @@ func domainXML(d *Driver, machineType string) (string, error) {
 			},
 		}
 	}
+	if virtiofsSupported(d.conn) == nil && len(d.SharedDirs) != 0 {
```

Comment:
it seems the test is failing because there's no real `d.conn` while running the test, as a workaround maybe nest the `virtiofsSupported` call under the len(d.SharedDirs) the tests doesn't configure shared dirs

```
github.com/code-ready/machine-driver-libvirt/pkg/libvirt.getBestGuestFromCaps(0x7f3a3c21c758?)
	/go/src/github.com/code-ready/machine-driver-libvirt/pkg/libvirt/libvirt.go:268 +0x2b
github.com/code-ready/machine-driver-libvirt/pkg/libvirt.virtiofsSupported(0xc000040ef0?)
	/go/src/github.com/code-ready/machine-driver-libvirt/pkg/libvirt/domain.go:162 +0x30
```